index.html

```diff
@@ -17,24 +25,50 @@
             <h2>
                 Static pages are the best
             </h2>
-        <nav>
-            <ul>
-                <li>GitHub</li>
-                <li>Twitter</li>
-                <li>About</li>
-                <li>Home</li>
-            </ul>
-        </nav>
-        </center>
+            </header>
+            <nav>
+                <ul>
+                    <li class="tab" data-content="home"><a href=#>Home</a></li>
+                    <li class="tab" data-content="about"><a href=#>About</a></li>
+                    <li><a href="http://www.something.com" target=_blank>Twitter</a></li>
+                    <li><a href="http://www.github.com/suhk" target=_blank>Github</a></li>
+                </ul>
+            </nav>
+            </center>
+
+        <div class="filter">
+            Sort by category:
+            <select id="category">
+                <option value="null">None</option>
+            </select>
+        </div>
+        <div class="filter">
+            Sort by author:
+            <select id="author">
+                <option value="null">None</option>
+            </select>
+        </div>
+    <div class="all">
+        <section id="home">
+            <main>
 
-    </header>
-    <main>
-        
-    </main>
+            </main>
+        </section>
+        <section id="about">
+            <div class="content" hidden>
```

Comment:
Having hidden here is keeping your about section hidden if you never remove it. I couldn't get your about section to work with it here.